modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/Index.java

```diff
@@ -74,7 +76,7 @@ final void close() {
         try {
             IgniteUtils.closeAll(storageByPartitionId.values().stream().map(index -> index::close));
         } catch (Exception e) {
-            throw new StorageException("Failed to close index storages: " + indexId, e);
+            throw new StorageException("Failed to close index storages: " + indexColumnFamily.indexId(), e);
```

Comment:
U can use `org.apache.ignite.internal.storage.StorageException#StorageException(int, java.lang.String, java.lang.Throwable, java.lang.Object...)`.

Maybe create method `Index#id()` to simplify?

Reply:
> U can use ...

yes, but what's the point? The current code is already quite laconic

> Maybe create method Index#id() to simplify?

To simplify what? `indexColumnFamily.indexId()`? It's not used that often

Reply:
`throw new StorageException("Failed to close index storages: {}", e, id());`
short and beauty

Reply:
Come on, it looks ok now as well....